package.json

```diff
@@ -133,6 +133,11 @@
     {
       "name": "Aras Abbasi",
       "email": "aras.abbasi@gmail.com"
+    },
+    {
+      "name": "Reidner Adnan",
+      "url": "https://github.com/reidn3r",
+      "email": "rdn.adn00@gmail.com"
```

Comment:
Please revert this. That section of the package.json is left for the collaborators (also listed in the README).

Reply:
Of course! I misunderstood. I apologize.